src/SimpleCalculator/obj/Debug/net6.0/SimpleCalculator.GeneratedMSBuildEditorConfig.editorconfig

```diff
@@ -7,4 +7,4 @@ build_property.InvariantGlobalization =
 build_property.PlatformNeutralAssembly = 

```

Comment:
\obj folder shouldn't be a part of code under a source control. It's autogenerated file, they can be easily created with a project building. Please updated gitingore file